flight/state_settings.py

```diff
@@ -0,0 +1,121 @@
+"""Class to contain setters and getters for settings in various flight states"""
+
+DEFAULT_WAYPOINTS: int = 14             # Total number of waypoints flown to in Auto. flight of SUAS2022
+DEFAULT_RUN_TITLE: str = "N/A"          # Title of flight mission for logging
+DEFAULT_RUN_DESCRIPTION: str = "N/A"    # Description of current flight mission for logging
+
+
+class StateSettings:
+    """
+    Initialize settings for state machine
+    Functions:
+        __init__: Sets preliminary values for SUAS overheads
+        simple_takeoff() -> None: Enables access to status of simple_takeoff
+                         -> bool: Setter to determine if testing takeoff procedure desired
+        num_waypoints() -> None: Enables access to number of waypoints in the competition
+                        -> int: Sets the number of waypoints in the competition
+        run_title() -> None: Enables access to set name of the current flight
+                    -> str: Sets the name of the current flight for logging
+        run_description() -> None: Enables access to description of current flight mission
+                          -> str: Sets the description of current flight mission
+    Member Variables:
+        __num_waypoints: Number of waypoints on flight plan
+        __run_title: Title of Competition
+        __run_description: Description for Competition
+    """
+    def __init__(self, takeoff_bool: bool = False, num_waypoints: int = DEFAULT_WAYPOINTS,
+                 title: str = DEFAULT_RUN_TITLE, description: str = DEFAULT_RUN_DESCRIPTION):
+        """Default constructor results in default settings
+        Args:
+            takeoff_bool: bool - determines if a simple takeoff procedure should be executed for testing
+            num_waypoints: int - Number of waypoints, extracted from SUAS mission plan
+            title: str - Title of current flight mission, for logging purposes
+            description: str - Description of current flight mission, for logging purposes
+        """
+        self.__simple_takeoff = takeoff_bool
+        self.__num_waypoints = num_waypoints
+        self.__run_title = title
+        self.__run_description = description
```

Comment:
I know it may seem redundant, but since this is the first time we're referencing these variable names, 
they should be typed as well